Dockerfile

```diff
@@ -1,15 +1,43 @@
 FROM --platform=$BUILDPLATFORM python:3.11-slim-bookworm
 ARG TARGETPLATFORM
 ARG BUILDPLATFORM
+ARG TARGETARCH
+RUN echo "I am running on $BUILDPLATFORM, building for $TARGETPLATFORM with $TARGETARCH"
 
 RUN <<EOF
-    set -e
-    apt-get update
-    apt-get install --assume-yes --no-install-recommends pipx dumb-init nodejs npm
+    set -eux
     groupadd --gid 1000 python
     useradd --uid 1000 --gid python --create-home --shell /bin/sh python
 EOF
 
+RUN <<EOF
+    set -eux
+    apt-get update
+    apt-get install --assume-yes --no-install-recommends pipx \
+    dumb-init \
+    nodejs \
+    npm \
+    unzip \
+    jq \
+    vim-tiny \
+    curl \
+    git \
+    wget
+
+    rm -rf /var/lib/apt/lists/*
+    apt-get clean
+EOF
+
+# TODO: fix
+RUN <<EOF
```

Comment:
do we need this to be separate from the previous RUN?

Reply:
we could potentially combine these, but it's so different of a process from simple apt-get install and the layer cache frequency is different. It's also not working yet. Maybe we can combine when I fix it?

Reply:
scratch that. It was working, I needed to force a pull with my testing. I combined them.